lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1286,6 +1299,19 @@ void addIntelFPGADecorationsForStructMember(
       E->addMemberDecorate(new SPIRVMemberDecorateMergeINTELAttr(
           E, MemberNumber, Name.str(), Direction.str()));
     } break;
+    case DecorationBankBitsINTEL: {
+      SmallVector<StringRef, 1> A;
+      StringRef(I.second).split(A, ',');
+      std::vector<SPIRVWord> Bits(A.size());
+      SPIRVWord Result;
+      for (size_t J = 0; J < A.size(); ++J) {
+        Result = 0;
+        A[J].getAsInteger(10, Result);
+        Bits[J] = Result;
+      }
```

Comment:
This piece of code looks identical to the non-member variant of this decoration. Can we factor it out?

Reply:
Sure